client/src/components/navbar/index.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component }from 'react';
+import ReactDOM from 'react-dom';
+
+import './style.css'    
+
+class Navbar extends Component{
+    constructor() {
+         super();
+         this.state={
+         activeTab: 'flights'
+        }
+
+        this.handleTabChange = this.handleTabChange.bind(this);
+    }
+
+    handleTabChange(activeTab) {
+        this.setState({
+         ...this.state,
+        activeTab
+        })
+    } 
+
+
+    render(){
+        return(
+      <div>    
+         <div className = "nav">
+         <div className ="user">
+         <a className="username" href={'#'} >username</a>
+         <img src={'/images/username.png'} />
+      </div>
+
+      <div>
+        <img className ="plane" src={'/images/plan.png'} />
+        </div>
+      </div>
+        <div className="sideDiv">
+        <ul>
```

Comment:
sideDiv looks like it's doing the same thing as the SideNav component (https://github.com/FACG4/nigerian-airlines/blob/6b14352977648f9a6156abf58df0249c423f8f4f/client/src/components/sidenav/index.js#L5)

They both seem to be the three buttons/links on the left side ("Flights", "Add Flight", "Update Flight").  Are both of them being used or could they be combined into one component?